universal-login-relayer/test/relayer/services/messages/PendingMessagesStore.ts

```diff
@@ -40,7 +40,7 @@ describe('UNIT: PendingMessagesStore', async () => {
 
   it('containSignature should return true if signarture already collected', async () => {
     pendingMessagesStore.add(messageHash, pendingMessage);
-    await pendingMessage.push(message);
+    await pendingMessage.collectedSignatures.push({signature: message.signature, key: '0x'});
```

Comment:
PendingMessages should use PendingMessagesStore to add new signature. `pendingMessagesStore.addSignature(messageHash)`

Reply:
✔️